src/scan.ts

```diff
@@ -122,3 +134,48 @@ async function scanDir(
     })
     .sort((a, b) => a.path.localeCompare(b.path));
 }
+
+export async function scanRouteMeta(
+  fullPath: string
+): Promise<ServerRouteMeta> {
+  const file = fs.readFileSync(fullPath, { encoding: "utf8", flag: "r" });
```

Comment:
Have you investigated how this performs for large projects with many routes and big files?

Reply:
(this is updated now)